src/main/java/com/google/devtools/build/lib/bazel/repository/downloader/HttpConnectorMultiplexer.java

```diff
@@ -61,7 +61,9 @@
   private static final int MAX_THREADS_PER_CONNECT = 2;
   private static final long FAILOVER_DELAY_MS = 2000;
   private static final ImmutableMap<String, String> REQUEST_HEADERS =
-      ImmutableMap.of("Accept-Encoding", "gzip");
+      ImmutableMap.of(
+          "Accept-Encoding", "gzip",
+          "User-Agent", "Bazel");
```

Comment:
It would be handy to encode the Bazel version in this, similar to how Browser do it (but hopefully with less compatibility-related garbage).